src/libraries/Native/Unix/System.Native/tests.c

```diff
@@ -0,0 +1,86 @@
+#include "pal_networking.h"
```

Comment:
License header missing?

Reply:
This file / commit will be reverted and won't be part of the final PR.
It's just for narrowing down the bug. These tests pass...